### DISCUSSION TITLE
how to do short ?

### DISCUSSION THREAD

**Discussion by ChainCoderX** [rodrigo-brito/ninjabot#93] [Q&A] [answered]
I want to sell asset then  buy asset, how to implement the feature ?

**Comment by rodrigo-brito** [comment]
Hi @wade-liwei. For now, we only implemented the Spot Market.
A good and easy way to "short" is trading ETFs. For example, in Binance we have BTCDOWN which is a leveraged token for BTC downtrend.

All these tokes are listed here: https://www.binance.com/en/leveraged-tokens/tokens/allTokens

**Comment by rodrigo-brito** [comment, marked as answer]
Hi @wade-liwei, now the bot supports Future Market: https://github.com/rodrigo-brito/ninjabot/blob/main/examples/futuremarket/futures.go
